packages/remix-react/components.tsx

```diff
@@ -177,13 +180,7 @@ export function RemixRouteError({ id }: { id: string }) {
       ErrorBoundary
     ) {
       // Internal framework-thrown ErrorResponses
-      return (
-        <RemixErrorBoundary
-          location={location}
-          component={ErrorBoundary!}
-          error={tError.error}
-        />
-      );
+      return <ErrorBoundary error={tError.error} />;
```

Comment:
Got rid of the Remix Error Boundary wrapper since we can just use on the RR one internally

Reply:
@jacob-ebey This is what I was talking about.  From my testing all seems fine here